lib/parse.js

```diff
@@ -68,7 +68,7 @@ module.exports = (input) => {
     device = _.chain(device)
       .split('\n')
       .filter((line) => {
-        return /^(\s\s-\s)?[a-z]+:/g.test(line);
+        return /^(\s\s[-\s]\s)?[a-zA-Z]+:/g.test(line);
```

Comment:
yes, that's exactly what I want :)
But this line `/^(\s\s-\s)?[a-z]+:/g.test(line);` removes volumeName, since it's has uppercase and has 4 spaces before